doc/contribute/index.rst

```diff
@@ -128,19 +128,35 @@ below and at http://developercertificate.org/.
         consistent with this project or the open source license(s)
         involved.
 
-DCO Sign-Off Methods
-====================
+DCO Sign-Off
+============
 
-The DCO requires a sign-off message in the following format appear on each
-commit in the pull request::
+The "sign-off" in the DCO is a "Signed-off-by:" line in each commit's log
+message. The Signed-off-by: line must be in the following format::
 
    Signed-off-by: Zephyrus Zephyr <zephyrus@zephyrproject.org>
 
-The DCO text can either be manually added to your commit body, or you can add
-either ``-s`` or ``--signoff`` to your usual Git commit commands. If you forget
-to add the sign-off you can also amend a previous commit with the sign-off by
-running ``git commit --amend -s``. If you've pushed your changes to GitHub
-already you'll need to force push your branch after this with ``git push -f``.
+For your commits, replace:
+
+- ``Zephyrus Zephyr`` with your legal name (pseudonyms, hacker handles, and the
```

Comment:
Can we use some common name placeholders here or not use place holders at all like before? Some legal names are more than just First and Last Name.

Reply:
> Some legal names are more than just First and Last Name.

This was already here before this PR and I think it's fine. Nowhere in the text do we say that the contributor's legal name is a given name and a surname.